RazorHtmlEmails.RazorClassLib/Views/RegistrationConfirmation.cshtml.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace RazorHtmlEmails.RazorClassLib.Views
```

Comment:
Maybe we can call this project **YesterdayApi.EmailViews** to be more consistent with others if you wanna keep that?

Reply:
Yeah, good call